spec/emoji_data_spec.rb

```diff
@@ -13,13 +13,19 @@
 
   describe ".all_doublebyte" do
     it "should return an array of all 21 known emoji chars with doublebyte encoding" do
-      EmojiData.all_doublebyte.count.should eq(21)
+      EmojiData.all_doublebyte.count.should eq(71)
```

Comment:
To be honest, I did not check that. It's most likely the new flags. I'm also not sure about the tests in general, as they're tightly coupled to the `emoji.json`.